README.md

```diff
@@ -38,8 +38,9 @@ You'll need to install FUSE on your system in order to use this project.
 For directions on how to install, see the [libfuse](https://github.com/libfuse/libfuse) github page.
 
 ```
-apt-get install libfuse-dev   # linux
-brew cask install osxfuse     # osx
+apt-get install libfuse-dev   # Linux (Ubuntu/Debian)
+brew cask install osxfuse     # Mac Os X
+sudo pacman -S fuse           # Linux (Manjaro/Arch)
```

Comment:
I'd keep the linux together and lower cased. I intentionally matched the casing of the go build process.

Reply:
What's the value of matching Go (or any) build process in comments, if it makes it less readable? Who are those comments for: machines to run, or people to read? camelCaseExistsForReason

Reply:
Not camel case, but I see your point. And you could have made it with a softer tone too.